packages/core/src/components/ConfirmationModal/ConfirmationModal.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { noop } from '../../common/helpers';
+import { Modal, Button } from '../../index';
+
+const ConfirmationModal = ({
+  show,
+  onCancel,
+  onConfirm,
+  title,
+  icon,
+  heading,
+  body,
+  cancelButtonText,
+  confirmButtonText,
+  className,
+  footer,
+  ...props
+}) => (
+  <Modal className={className} show={show} onHide={onCancel} {...props}>
+    <Modal.Header>
+      <Modal.CloseButton onClick={onCancel} />
+      <Modal.Title>{title}</Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
+      <div className="pf-confirm-modal-body">
+        {icon && <span className="pf-confirm-modal-icon">{icon}</span>}
+        <span className="pf-confirm-modal-content">
+          <span className="spacer" />
```

Comment:
You can remove the `.spacer` spans and set `justify-content: center` on the `.pf-confirmation-modal-content` class.